VMBackup/main/workloadPatch/WorkloadPatch.py

```diff
@@ -143,6 +143,19 @@ def preMaster(self):
                 sleep(2)
             self.timeoutDaemon()
             self.logger.log("WorkloadPatch: Pre- Exiting pre mode for master")
+        elif 'postgres' in self.name.lower():
```

Comment:
if possible please refactor each workload specific code to workload specific utils files so that code looks modular and readable.

Reply:
https://microsoftapc-my.sharepoint.com/personal/pullabhk_microsoft_com/_layouts/OneNote.aspx?id=%2Fpersonal%2Fpullabhk_microsoft_com%2FDocuments%2FOracle%20DB%20backup&wd=target%28Worklist.one%7C1CFB48C1-0FC1-432C-9985-45A6301C7F4A%2FLinux%20App%20Consistency%7C9292B7E3-3DB5-43A6-AF87-2829A6434340%2F%29
onenote:https://microsoftapc-my.sharepoint.com/personal/pullabhk_microsoft_com/Documents/Oracle%20DB%20backup/Worklist.one#Linux%20App%20Consistency&section-id={1CFB48C1-0FC1-432C-9985-45A6301C7F4A}&page-id={9292B7E3-3DB5-43A6-AF87-2829A6434340}&end